README.md

```diff
@@ -1,8 +1,24 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## [Game Dev Grit Podcast Website](https://rachel-blue.github.io/GameDevGrit/)
 
-## Available Scripts
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-In the project directory, you can run:
+## How to run the project locally
+
+## `Installing Dependencies`
```

Comment:
Should be  `###` since it's a nested header.

Reply:
This makes no sense, it should simply tell the user to do the following.

1. Install Node.js
2. Clone the repo
3. Run `npm install` in the root
4. Run 'npm start` to view the page

See here for a live but much more complex example I've written https://github.com/ashblue/oyster-package-generator#getting-started